packages/shared/src/contexts/PaymentContext.tsx

```diff
@@ -149,7 +150,7 @@ export const PaymentContextProvider = ({
     });
   }, [paddle, planTypes, geo?.region]);
 
-  const { data: productPrices } = useQuery({
+  const { data: productPrices, isLoading: isPricesPending } = useQuery({
```

Comment:
it is now `isPending` if you want to wait for prices to be available

Reply:
Crazy how they changed the variables behaved.